webpack.config.js

```diff
@@ -174,6 +175,25 @@ const devConfig = {
     },
 };
 
+const devMv3Config = {
+    ...commonConfig,
+    entry: {
+        ...commonEntryFiles,
+        detailsView: [...reactDevtoolsEntryFiles, ...commonEntryFiles.detailsView],
+        popup: [...reactDevtoolsEntryFiles, ...commonEntryFiles.popup],
+    },
+    name: 'dev-mv3',
+    mode: 'development',
+    devtool: 'inline-cheap-source-map',
```

Comment:
> I experimented with both inline-cheap-source-map and inline-source-map, which is slightly faster. I opted for the slower option because the difference was only about 5 seconds per build.

I also tested this and my findings matched yours: On my machine, `inline-source-map` was faster than `inline-cheap-source-map` (in 3 trials with 1 warmup iteration each, `yarn build:dev:mv3` took `51±2s` for the former and `57±8s` for the latter on my Surface Laptop 3 running win11).

[Webpack's documentation](https://webpack.js.org/configuration/devtool/) suggests that the only functional difference between them should be that the `cheap` variant will have a slightly less accurate source map (it maps only line numbers, where the non-`cheap` variant maps both line and column numbers). Did you find some other reason to prefer the `cheap` option? It seems like the non-cheap one is both faster and more accurate, so I think it would make more sense to use that.

(I think we found similar timings in the past when we last benchmarked `eval-source-map` vs `eval-cheap-source-map`, and that's why the existing `dev` config uses the former)

Reply:
Ironically, I intended to use `inline-source-map`, but then forgot to make the change--I should know better than to write PR's in a brain fog ☹️ 

Fixed to use `inline-source-map`